src/overlays/actors/ovl_Boss_Va/z_boss_va.c

```diff
@@ -607,7 +607,7 @@ void BossVa_Init(Actor* thisx, PlayState* play2) {
             break;
         default:
             this->actor.flags |= ACTOR_FLAG_24;
-            SkelAnime_Init(play, &this->skelAnime, &gBarinadeBariSkel, &gBarinadeBariAnim, NULL, NULL, 0);
+            SkelAnime_Init(play, &this->skelAnime, &gBarinadeBariSkel.sh, &gBarinadeBariAnim, NULL, NULL, 0);
             this->actor.shape.yOffset = 400.0f;
```

Comment:
these cases of using the `sh` field feel weird, wasnt aware there were cases of the wrong skelanime function being used. I guess if it ends up working in practice its fine

Reply:
An alternative could be: `(SkeletonHeader*)&gBarinadeBariSkel`